extensions/arc/src/ui/dashboards/miaa/miaaDashboardOverviewPage.ts

```diff
@@ -86,8 +83,11 @@ export class MiaaDashboardOverviewPage extends DashboardPage {
 
 		this._kibanaLink = this.modelView.modelBuilder.hyperlink().component();
 		this._grafanaLink = this.modelView.modelBuilder.hyperlink().component();
+		this.handleEndpointsUpdated();
```

Comment:
@Charles-Gagnon  For reasons I don't understand, it matters that the component has data before it's added to another component.  e.g. if these endpoints are updated after they're added to the loading components, they don't show data.

This applied to all the Postgres components too, which is why they're implemented to have data when they're first constructed.

More work is needed to comply with this requirement.  I'm hoping you can take over this PR.  There's a property for the username that isn't set until handleDatabasesUpdated is called below.  But it should be set before the properties container is added.

In my environment the call to fetch databases is slower than the controller calls, so that callback isn't missed by the UI.  (In my gif you can see the username and databases load fine).  That event ends up refreshing the properties, which made me wonder why the properties container seemed immune to this requirement.  But you can remove the databases callback to see that it applies there as well.